MakeYourCake.Services.Api/Brokers/Loggings/LoggingBroker.cs

```diff
@@ -0,0 +1,35 @@
+﻿// -----------------------------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// -----------------------------------------------------------------------
+
+using Microsoft.Extensions.Logging;
+using System;
```

Comment:
Run code cleanup

Reply:
Fixed

Reply:
I still see System not being sorted before other libraries.

Reply:
Updated